static-site/content/featured-analyses.yaml

```diff
@@ -4,31 +4,36 @@
 # - When adding or modifying descriptions, ensure it takes up at most two lines
 #   when rendered on the splash page.
 
-- name: Mpox in the DRC
-  description: INRB analysis of ongoing mpox clade I outbreak in the DRC
-  img: mpox_drc.png
-  url: /groups/inrb-mpox/clade-I
-
-- name: Mpox
-  description: Evolution and spread of mpox clades I and II
-  img: mpox_tree.png
-  url: /mpox/all-clades
-
 - name: SARS-CoV-2
   description: Ongoing evolution and spread of SARS-CoV-2
   img: ncov_freq.png
   url: /ncov/gisaid/global/6m
 
+- name: Mpox in the DRC
+  description: INRB analysis of ongoing mpox clade I outbreak in the DRC
+  img: mpox_drc.png
+  url: /groups/inrb-mpox/clade-I
+
 - name: H5N1 cattle outbreak
   description: Analysis of the ongoing influenza A/H5N1 cattle outbreak
   img: h5n1_cattle_outbreak_tree.png
   url: /avian-flu/h5n1-cattle-outbreak/genome
 
+- name: HPAI outbreaks
+  description: "Highly pathogenic avian influenza (HPAI) in North America"
```

Comment:
Description overflows at my default zoom level.

![image](https://github.com/user-attachments/assets/ea574fb6-ab70-495f-b7aa-0273c604625d)

I think the easiest fix is to tweak the margin slightly: fe5e26b10054033f8f1a4f748e4983ce78d4bd58

Reply:
what browser and zoom level? I cannot reproduce that with Firefox on MacOS at any zoom level I've tried. 

i would really rather not get into the business of single pixel tweaks to margins to fix layout issues; that way lies madness.

Reply:
Zoom level <=100%, reproduced on Vivaldi (Chromium), Google Chrome, and Safari

Reply:
removed " (HPAI) " from description